### PR TITLE
Release 27.6

### PR DESCRIPTION
**Pull request by lucasmoura**

### CODE REVIEW

**Comment by lucasmoura** [comment]
@CalvoM this is the PR we put for a release.
As you can see, we bring the commits from 'main` that we want to include in the release while also creating a new `debian/changelog` entry and bumping the version string

We test this branch by comparing it to main to see if we are not missing anything.
We also build a package from this branch for every series we support to see if everything is working as expected.

Please let me know if you have any other questions regarding it

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,17 @@
+ubuntu-advantage-tools (27.6~22.04.1) jammy; urgency=medium
+
+  * d/tools.postinst:
+    - make log files world readable
+  * New upstream release 27.6 (LP: #1958556)
+    - cli: only go for resources on explicit help calls
```
Comment (orndorffgrant) [inline debian/changelog:6]: "go for resources" makes sense to me but likely won't for someone outside the project
```suggestion
    - cli: only request available resources from contract server when necessary
```

File: debian/changelog [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+    - fips:
+      + allow enabling FIPS on focal clouds
+      + update prompt messages
+    - jobs: disable jobs after attach/auto-attach
```
Comment (orndorffgrant) [inline debian/changelog:10]: Specifically the license-check job
```suggestion
    - jobs: disable license-check job on GCP after attach
```

**Review by orndorffgrant — COMMENTED** [review]
Two small changelog suggestions

File: debian/changelog (hunk 2 above)
Comment (lucasmoura) [reply debian/changelog:10]: Just modified the message a little bit for size constraints. Let me know what you think

**Review by orndorffgrant — APPROVED** [review]
Looks great! Thank you

**Comment by CalvoM** [comment]
Thanks for tagging me on this.

**Pull request merged**